Makefile

```diff
@@ -682,7 +682,7 @@ SPEED_OPTIMISED_SRC = \
             telemetry/mavlink.c \
             telemetry/esc_telemetry.c \
 
-SIZE_OPTIMISED_SRC = \
+SIZE_OPTIMISE_CANDIDATES_SRC := \
```

Comment:
Well it's only specifically optimised for size for F3. We should make that clear lest people add stuff to this list in an effort to make F1 targets fit.